app/config/sources.inc.php

```diff
@@ -392,6 +392,10 @@
         'priority'          => 1,
         'supported_locales' => $mozillaorg,
     ],
+    'mozorg/500.lang' => [
+        'priority'          => 1,
```

Comment:
I would leave this as priority 3 (remove the line), it's not a page they should see…